### PR TITLE
Add project-not-found error

### PR DESCRIPTION
**Pull request by iannbing**
## Description

Currently Orechest switches to an existing project if the given project UUID is invalid. This PR adds an alert if the said case occurs.

Screenshot 2022-04-25 at 13 37 10

## Checklist
- [x] The PR branch is set up to merge into `dev` instead of `master`.

### CODE REVIEW

File: services/orchest-webserver/client/src/contexts/__tests__/ProjectsContext.test.tsx [hunk 1]
```diff
@@ -32,6 +32,13 @@ describe("useProjectsContext", () => {
     // Test case starts
 
     act(() => {
+      // Temporarily not mocking the payload fully.
+      result.current.dispatch({
+        // @ts-ignore
+        type: "SET_PROJECTS",
+        // @ts-ignore
+        payload: [{ uuid: MOCK_PROJECT_ID_1 }, { uuid: MOCK_PROJECT_ID_2 }],
+      });
```
Comment (iannbing) [inline services/orchest-webserver/client/src/contexts/__tests__/ProjectsContext.test.tsx:41]: This mock data will be added in another PR #893.

File: services/orchest-webserver/client/src/components/ProjectSelector.tsx [hunk 2]
```diff
@@ -96,18 +110,30 @@ export const ProjectSelector = () => {
...
+        validateProjectUuid(projectUuidFromRoute, state.projects);
+
+      if (projectUuidFromRoute && !validProjectUuid) {
+        setAlert(
+          "Project not found",
+          <Stack direction="column" spacing={2}>
+            <Box>
+              {`Project with the given uuid `}
+              <Code>{projectUuidFromRoute}</Code>
+              {` is not found. This project might have been deleted, or you might have had a wrong URL.`}
```
Comment (fruttasecca) [inline services/orchest-webserver/client/src/components/ProjectSelector.tsx:125]: Might need a "The" in front of project at the beginning, perhaps we could make the first sentence shorter altogheter, like "Couldn't find project <uuid>."

**Review by fruttasecca — APPROVED** [review]
lgtm :)
